subprojects/core/src/main/java/org/gradle/api/internal/tasks/DefaultTaskContainer.java

```diff
@@ -656,14 +656,14 @@ private IllegalStateException createIllegalStateException() {
     @Deprecated
     @Override
     public boolean add(Task o) {
-        DeprecationLogger.nagUserOfReplacedMethodWithoutRemoval("add()", "create() or register()");
+        DeprecationLogger.nagUserOfReplacedMethodWithoutCustomRemoval("add()", "create() or register()", "This method will cause an error in Gradle 6.0.");
```

Comment:
We should probably have the same wording as the default removal method. Why did you choose this one instead?

Reply:
This method won’t actually be removed, as it’s part of an interface we have to implement.